ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/utils/Logging.kt

```diff
@@ -47,7 +51,19 @@ internal class TestLogger(private vararg val expectedLog: String) : Logger {
                 continue
             }
 
-            assertEquals(expected, actual)
+            if (expected != actual) {
+                println(">>> Expected log:")
```

Comment:
What are all those print statements for? Why not use asserts?

Reply:
Because it's a log message. 
Actually, I can use `fail(message)` here, but it will look nasty.

Reply:
It looks like a hack to use print statements in tests.